tools/refresh.sh

```diff
@@ -285,15 +286,17 @@ for CONFIG in ${CONFIGS}; do
 
   # Restore any previous .config and Make.defs files
 
+  if [ -e SAVEMake.defs ]; then
+    mv SAVEMake.defs Make.defs || \
+      { echo "ERROR: Failed to move SAVEMake.defs to Make.defs"; exit 1; }
+  fi
+
   if [ -e SAVEconfig ]; then
     mv SAVEconfig .config || \
       { echo "ERROR: Failed to move SAVEconfig to .config"; exit 1; }
+    ${TOPDIR}/tools/sethost.sh 1>/dev/null
```

Comment:
Should we change like other place:
 if [ "X${debug}" = "Xy" ]; then
   tools/sethost.sh V=1
 else
   tools/sethost.sh 1>/dev/null
  fi